src/redux/flightcontroller/flightControllerSlice.ts

```diff
@@ -0,0 +1,43 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface FlightControlState {
+  settings: {
+    isEnabled: boolean;
+    inputScaleFactor: number;
+  };
+}
```

Comment:
If you want to make it clear that these parameters are for the settings of the flight controller, maybe the entire state should be renamed to "flightControllerSettings" ?  That would also make it clearer that this state is not responsible for sending the input information for the flight controller, for example. 

Note that this would also include renaming the file to flightControllerSettingsSlice.ts

Reply:
My reasoning was that since we didn't have a `flightControllerSlice` and this new slice lives in the `flightcontroller` folder I wanted to be explicit about which part of the state is for settings, in case we decide to add other states to the flightcontroller in the future.

Reply:
Changing the state but keeping the name of the file and slice